<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateIconDialog</class>
 <widget class="QDialog" name="CreateIconDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icon</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelX">
           <property name="text">
            <string>X :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelY">
           <property name="text">
            <string>Y :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCoords">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelDate">
         <property name="text">
          <string>Date : </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDate">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelRotation">
         <property name="text">
          <string>Rotation : </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRotation">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonCoords</tabstop>
  <tabstop>buttonDate</tabstop>
  <tabstop>buttonRotation</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateIconDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateIconDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCoords</sender>
   <signal>clicked()</signal>
   <receiver>CreateIconDialog</receiver>
   <slot>onButtonCoordsEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDate</sender>
   <signal>clicked()</signal>
   <receiver>CreateIconDialog</receiver>
   <slot>onButtonDateEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRotation</sender>
   <signal>clicked()</signal>
   <receiver>CreateIconDialog</receiver>
   <slot>onButtonRotationEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonCoordsEdit()</slot>
  <slot>onButtonDateEdit()</slot>
  <slot>onButtonRotationEdit()</slot>
 </slots>
</ui>
